.travis.yml

```diff
@@ -0,0 +1,50 @@
+language: python
+python:
+  - 3.9
+  - 3.10
+git:
+  depth: false
+
+# Ubuntu 20.04 (Focal) LTS
+os: linux
+dist: focal
```

Comment:
Currently, `ark-analysis` builds `xenial` on Travis and `bullseye` on Docker. We'll need to decide on a standard Linux distro we'll want to use across our repos and Docker builds.

Reply:
It looks like not yet, `bullseye` is fairly recent. Perhaps `focal` would be best for `ark-analysis` too, we'll talk at tomorrow's standup.